modules/core/03-connection/keeper/verify.go

```diff
@@ -24,7 +24,7 @@ func (k Keeper) VerifyClientState(
 	clientState exported.ClientState,
 ) error {
 	clientID := connection.GetClientID()
-	targetClient, clientStore, err := k.getClientStateAndKVStore(ctx, clientID)
+	targetClient, clientStore, err := k.getClientStateAndVerificationStore(ctx, clientID)
```

Comment:
I don't feel super strongly about it. Happy to go with whatever the team prefers

Reply:
I think I have no preference since `getVerificationStore` would take in the clientID and the client state. Maybe we rename to something like `getVerificationArgs` and move the status check in this helper function as well? Any other naming ideas? cc @chatton

Reply:
I actually quite like what is in this PR already `getClientStateAndVerificationStore`, it's super explicit.

Reply:
Going to leave as is since I don't think it's worth spending too much time on this